CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Changes to SUNDIALS in release X.Y.Z
 
+Fixed the runtime library installation path for windows systems. This fix changes the
```

Comment:
```suggestion
Fixed the runtime library installation path for windows systems. This fix changes the 
default library installation path from `CMAKE_INSTALL_PREFIX/CMAKE_INSTALL_LIBDIR` to
`CMAKE_INSTALL_PREFIX/CMAKE_INSTALL_BINDIR`.
```

Reply:
Markdown formatting should use single backticks for inline code. Please also add this to `docs/shared/RecentChanges.rst` as well (which expects the double backticks).